StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/PaymentMethodTypeCollectionView.swift

```diff
@@ -214,8 +214,17 @@ extension PaymentMethodTypeCollectionView {
         private lazy var promoBadge: PromoBadgeView = {
             PromoBadgeView(appearance: appearance, tinyMode: true)
         }()
-        private lazy var shadowRoundedRectangle: ShadowedRoundedRectangle = {
-            return ShadowedRoundedRectangle(appearance: appearance)
+        private lazy var selectableRectangle: SelectableRectangle = {
+            #if !os(visionOS)
```

Comment:
sorry i'm forgetting, why do we need this again?

Reply:
Should we make the LiquidGlassDetector.isEnabled return false on visionOS? Then we dont need to add these everywhere.